### ISSUE TITLE
ESC should interrupt download when connecting to server

### ISSUE THREAD

**Issue by RamchandraApte** [Unvanquished/Unvanquished#911]
When connecting to a server, downloads (of maps, etc) can take a long time, so ESC should interrupt the download. Using the console and typing disconnect works, so I think it should be pretty easy to make ESC disconnect.

**Comment by mbasaglia** [comment]
You can bind it if you want.

**Comment by DolceTriade** [comment]
Esc already opens up the inagme menu. Also, binds aren't active at this stage. It is a simple fix however: onKeyDown="function (event, ...) if event.params.key == rocket.key.ESCAPE then Cmd.Exec('disconnect') end end"

**Comment by mbasaglia** [comment]
Would that interfere with Shift+Esc?

**Comment by DolceTriade** [comment]
Nah, this escape would only be handled when this particular menu is open. Shift-Esc is handled at the engine level, so this wouldn't interfere.
